helpers/filterSeats.js

```diff
@@ -60,19 +69,73 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections() {
+      // add all section names to an array
+      _.forEach(bandColours, function(bands, section) {
+        bestSections.push(section);
+      });
+      // find the best section
+      bestSections = _.last(bestSections); // just Stalls
+    }
+
+    getBestSections();
+
+    // compare prices
+    function chosenForYou(ticketRate, ticketRateSection) {
+      // ticketRateSection
+      var ticketRateId = ticketRate.ids;
+      var ticketRateColour = ticketRate.colour;
+      var ticketRateColourRank = ticketRate.colourRank;
+      var ticketRatePrice = ticketRates[ticketRateId].grossPrice;
+      // check if current ticketRate exists in
+      if (bestSections === ticketRateSection && _.last(ticketRateColourRank)) {
+        // check if price is cheaper or exists
+        if (ticketRatePrice < currentPrice || !currentPrice) {
+          // assign ticket to object
+          bestPrice = {
+            id: ticketRateId,
+            price: ticketRatePrice,
+            colour: ticketRateColour,
+            colourRank: ticketRateColourRank
+          };
+          currentPrice = ticketRatePrice;
+        }
+      }
+      // get the total amount of tickets set to object
+      var totalTickets = Object.keys(bestPrice).length;
+      // check if any tickets are set
+      if (totalTickets > 0) {
+        // sort best tickets by price then by colour
+        bestPrice = _.sortBy(bestPrice, ['price', 'colourRank']);
+        // find out if the top ticket is gold
+        if (bestPrice[3] === 2) {
```

Comment:
`bestPrice` is an object but it might be worth reworking it to populate it as an array.